config/Makefile.msvc64

```diff
@@ -115,8 +115,8 @@ LDFLAGS=/ENTRY:wmainCRTStartup
 ### Libraries needed
 #EXTRALIBS=bufferoverflowu.lib  # for the old PSDK compiler only
 EXTRALIBS=
```

Comment:
The EXTRALIBS variable is now dead so you could remove those two lines entirely.  Or are there other uses of EXTRALIBS in other Makefiles?

Reply:
Note: we discussed in the other PR with those commits, https://github.com/ocaml/ocaml/pull/1408#discussion_r143356895, and it turns out EXTRALIBS is used in `stdlib/Makefile`.

Reply:
EXTRALIBS should be dead, but I didn't quite have the guts to remove it as part of this (it should really be that the person configuring OCaml adds them to the appropriate place in BYTECCLIBS, etc.). What I did have the guts (!!) to amend was the inconsistency between msvc64 and mingw64 where EXTRALIBS is concerned and also that byterun/Makefile manually specifies the libraries to link rather than correctly using BYTECCLIBS.

Reply:
OK, let's keep EXTRALIBS around for the moment.